static/js/narrow.js

```diff
@@ -256,7 +256,13 @@ exports.activate = function (raw_operators, opts) {
     $('#search_query').val(Filter.unparse(operators));
     search.update_button_visibility();
 
-    compose_actions.on_narrow(opts);
+    // If the trigger was due to compose_hotkey we
+    // keep the compose box in its previous state,
+    if (opts.trigger === "compose_hotkey") {
+        compose_actions.on_compose_narrow(opts);
+    } else {
+        compose_actions.on_narrow(opts);
+    }
```

Comment:
Can'y we avoid this ugly if statement by just making `compose_actions.on_narrow` handle this case (either through a new option or checking the trigger?)

Reply:
Handled.